src/ios/CDVInAppBrowser.m

```diff
@@ -1117,7 +1208,7 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interface
 
 @implementation CDVInAppBrowserOptions
 
-- (id)init
+- (id)initWithOptions:(NSString*)optionStrings
```

Comment:
You are not using optionStrings at all in this method. The change is not necessary. Revert it back to how it used to be